docs/development/testguide.rst

```diff
@@ -987,73 +966,3 @@ from `astropy-benchmarks <https://github.com/astropy/astropy-benchmarks/>`_.
 It is important to note that these benchmarks can be flaky as they run on
 virtual machines (and thus shared hardware) but they should give a general
 idea of the performance impact of a pull request.
-
-.. _pytest-plugins:
-
-Pytest Plugins
-**************
-
-The following ``pytest`` plugins are maintained and used by Astropy. They are
-included as dependencies to the ``pytest-astropy`` package, which is now
-required for testing Astropy. More information on all of the  plugins provided
-by the ``pytest-astropy`` package (including dependencies not maintained by
-Astropy) can be found `here <https://github.com/astropy/pytest-astropy>`__.
-
-.. _remotedata-plugin:
-
-pytest-remotedata
-=================
-
-The |pytest-remotedata| plugin allows developers to control whether to run
-tests that access data from the internet. The plugin provides two decorators
-that can be used to mark individual test functions or entire test classes:
-
-* ``@pytest.mark.remote_data`` for tests that require data from the internet
```

Comment:
The `remote_data` marker is already documented in the "Working with data files" section.